x/whitelist/keeper/keeper.go

```diff
@@ -115,3 +115,28 @@ func (k keeper) GetDenomWhitelist(ctx sdk.Context) types.DenomWhitelist {
 
 	return whitelist
 }
+
+func (k keeper) GetNormalizationFactor(ctx sdk.Context, denom string) (sdk.Dec, bool) {
```

Comment:
GetNormalizationFactor is a CLP function so should be implemented there instead of on whitelist keeper

Reply:
Moved it , but the I was under the impression GetNormalizationFactor might be moved to the peggy bridge and used in ibc_sifchain .
 CLP in general is a mathematical module , it should not be dealing with normlization etc .